docs/content/en/docs/user-guide/command-line-tool.md

```diff
@@ -16,11 +16,12 @@ You can use pipectl to add and sync applications, wait for a deployment status.
 1. Download the appropriate version for your platform from [PipeCD Releases](https://github.com/pipe-cd/pipe/releases).
 
     We recommend using the latest version of pipectl to avoid unforeseen issues.
-    Please set `{VERSION}` to the same format like `v0.9.15`.
-    And `{OS}` can be replaced with either `linux` or `darwin`.
+    Run the following script:
 
     ``` console
-    curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/{VERSION}/pipectl_{VERSION}_{OS}_amd64
+    OS="darwin" # or "linux"
+    URL=$(curl -s https://api.github.com/repos/pipe-cd/pipe/releases/latest | grep browser_download_url | grep pipectl | grep "${OS}" | cut -d : -f 2,3 | tr -d '" ')
```

Comment:
This pipeline does not look smart 😅  What do you think?

Reply:
There is a quick way to get the latest version in the doc is using this block.

```
{{< blocks/latest_version >}}
```

So how about this?

``` console
OS="darwin" # or "linux"
curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/{{< blocks/latest_version >}}/pipectl_{{< blocks/latest_version >}}_${OS}_amd64
```

Reply:
🆒 ! I will fix it 👍

Reply:
@chaspy Could you fix this?

Reply:
Fixed 🙏 
https://github.com/pipe-cd/pipe/pull/2856/commits/99286b6a80314936a8749eef4d8692671e42eaae